xml/mysql_banners.xml

```diff
@@ -903,8 +904,132 @@
     <param pos="0" name="os.version" value="13.10"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:13.10"/>
   </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~buster(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB on Debian 10 (Buster)</description>
+    <example service.version="10.3.20">5.5.5-10.3.20-MariaDB-1:10.3.20+maria~buster-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Debian"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="10.03"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:debian_linux:10.03"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~yakkety(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB on Ubuntu 16.10 (Yakkety)</description>
+    <example service.version="10.1.26">5.5.5-10.1.26-MariaDB-1~yakkety</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="16.10"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:13.10"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB-1~yakkety-(?:wsrep)?$" flags="REG_ICASE">
+    <description>MariaDB-wsrep on Ubuntu 16.10 (Yakkety)</description>
+    <example service.version="10.0.28">5.5.5-10.0.28-MariaDB-1~yakkety-wsrep</example>
```

Comment:
Since we're not indicating the use of the `wsrep` patch in the values we emit, shouldn't this just be combined with the fingerprint above?  Alternatively you can do what was done further down and indicate that it's the Galera cluster edition.